eng/tools/eng-package-utils/update-samples.js

```diff
@@ -37,6 +37,12 @@ async function main(repoRoot, artifactName) {
     console.error(`Package is not found in rush.json for artifact ${artifactName}`);
     return;
   }
+
+  if (targetPackage.versionPolicyName == "management") {
+    console.error(`Skipping update samples for management package ${artifactName}`);
```

Comment:
Did we want that do be an error or just a message?

Reply:
I prefer just a message. @praveenkuttappan Should we change it back?

Reply:
looks like he did that with https://github.com/Azure/azure-sdk-for-js/pull/17454